### PR TITLE
(1204): Clarified Python Project Procfile Requirements

### PR DESCRIPTION
**Pull request by ndneighbor**
_Say that 5 times, 5 times as fast_

Thanks to Discord user: andreawthaderp#3031 for catching this issue. Added copy to clarify the Python Procfile requirements.

Comments from the Author
- Might be good to call out that we look for `web:` in the file but I will spend some time thinking about the bigger picture of docs before including that.

### CODE REVIEW

File: src/pages/deployment/builds.md [hunk 1]
```diff
@@ -48,6 +48,8 @@ The [Python buildpack](https://github.com/heroku/heroku-buildpack-python)
 detects if your build is Python by looking for a `requirements.txt` file. If
 found, dependencies will be installed using `pip`.
 
+Please include a [Procfile](/deployment/builds#procfile) in the root folder of your repository. If no [Procfile](/deployment/builds#procfile) is found, your deploy might fail to start.
```
Comment (JakeCooper) [inline src/pages/deployment/builds.md:51]: Might want to replace it with the actual error

That way when people are searching for that specific error, whether docs or SEO stuff, it'll come up

**Review by JakeCooper — APPROVED** [review]

File: src/pages/deployment/builds.md (hunk 1 above)
Comment (ndneighbor) [reply src/pages/deployment/builds.md]: Can you check now to see if it's preferable?

File: src/pages/deployment/builds.md [hunk 2]
```diff
@@ -92,8 +98,7 @@ process1: command
 process2: command
```
Comment (coffee-cup) [inline src/pages/deployment/builds.md:98]: Might want to remove this bit too since we don't support multiple processes

Comment (ndneighbor) [reply src/pages/deployment/builds.md:98]: Lemme fix that

File: src/pages/deployment/builds.md [hunk 3]
```diff
@@ -94,11 +94,10 @@ the deployment starts.
 A Procfile is in the format of
 
 ```
-process1: command
-process2: command
+process: command
```
Comment (coffee-cup) [inline src/pages/deployment/builds.md:97]: very minor nit: Use `web` here since we should push users in that direction since it works for web servers

**Review by coffee-cup — APPROVED** [review]

File: src/pages/deployment/builds.md (hunk 1 above)
Comment (JakeCooper) [reply src/pages/deployment/builds.md]: What do you mean?

File: src/pages/deployment/builds.md (hunk 3 above)
Comment (ndneighbor) [reply src/pages/deployment/builds.md:97]: Unsure if we should do that as it's explaining a procfile, also, the web Procfile sample is directly after

**Pull request merged**